webpack.dev.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+const sassPaths = require('@nypl/design-toolkit').includePaths
```

Comment:
This is only to be able to import the design toolkit easily in the main scss file.